src/content/doc-surrealql/functions/database/api.mdx

```diff
@@ -0,0 +1,264 @@
+---
+sidebar_position: 2
+sidebar_label: API functions
+title: API functions | SurrealQL
+description: These functions can be used with the DEFINE API or DEFINE CONFIG statements.
+---
+
+# API functions
+
+> [!NOTE]
+> API middleware functions such as `api::req::max_body` cannot be used in an ad-hoc manner, such as via Surrealist or the CLI. Instead, they are passed in to a [`DEFINE API`](/docs/surrealql/statements/define/api) or [`DEFINE CONFIG API`](/docs/surrealql/statements/define/config) statement to be used as middleware when a request is received. The only function that can be run at any time is [`api::invoke`](#apiinvoke) which is used to test API endpoints.
+
```

Comment:
```suggestion
> [!CAUTION]
> Currently, this is an experimental feature as such, it may be subject to breaking changes and may present unidentified security issues. Do not rely on this feature in production applications. To enable this, set the `SURREAL_CAPS_ALLOW_EXPERIMENTAL` [environment variable](/docs/surrealdb/cli/start/#experimental-capabilities) to `define_api`.
```